### ISSUE TITLE
Bootloop after formatting /data with LineageOS 17.1 on XZ2 Compact

### ISSUE THREAD

**Issue by K4sum1** [lineageos4microg/android_vendor_partner_gms#26]
I've been compiling LineageOS 17.1 for my XZ2 Compact, and if I format /data in LineageOS Recovery with a MicroG compiled ROM installed, it causes the device to bootloop. It works fine if I compile without MicroG. I can't seem to get a logcat, adb logcat gives me an inaccessible or not found error.

**Comment by petefoth** [comment]
Sorry 17.1 is very old - latest XZ2 Compact code is 20.0. I'm not sure 17.1 will even build: we certainly can't provide any support for it. Does the latest ROM from [here](https://download.lineage.microg.org/xz2c/) run OK?

**Comment by K4sum1** [comment]
> Sorry 17.1 is very old - latest XZ2 Compact code is 20.0. I'm not sure 17.1 will even build: we certainly can't provide any support for it. Does the latest ROM from [here](https://download.lineage.microg.org/xz2c/) run OK?

Gives me a "Package is for product xz2c but expected H8324" error. My device is a H8314 XZ2C.

**Comment by petefoth** [comment]
Does the latest standard LineageOS ROM from [here](https://download.lineageos.org/devices/xz2c/builds) work?

**Comment by K4sum1** [comment]
> Does the latest standard LineageOS ROM from [here](https://download.lineageos.org/devices/xz2c/builds) work?

Yes, that installs fine and wiping /data is fine with it too. I tested it before I started building anything.

**Comment by petefoth** [comment]
Sorry, I'm confused. Is your built 17.1 ROM that doesn't work, or our 20.0 ROM, or your own built 20.0 ROM?

**Comment by K4sum1** [comment]
> Sorry, I'm confused. Is your built 17.1 ROM that doesn't work, or our 20.0 ROM, or your own built 20.0 ROM?

17.1 custom built stock:
Works fine

17.1 custom built microG:
Works, however bootloops if I wipe /data

20.0 stock:
Works fine

20.0 microG:
Doesn't install

I have not tried building 20.0

**Comment by petefoth** [comment]
> 20.0 microG: Doesn't install

Did you format data before installing?

How did you try to install?

**Comment by K4sum1** [comment]
> Did you format data before installing?
> 
> How did you try to install?

I booted into recovery, did adb sideload, and got that error. I didn't try formatting data before install though. I usually do that in TWRP, but I guess I just forgot since I'm not used to LineageOS recovery. Will try that.

**Comment by petefoth** [comment]
That makes sense. You need to format data when switching from 17.1 to 20.0

**Comment by K4sum1** [comment]
So I installed it, and it seems to work fine. Is there anything I can try to fix my ROM?

**Comment by petefoth** [comment]
> Is there anything I can try to fix my ROM?

17.1 ROM? No sorry there's nothing I can think of. We're not in a position to support building old versions when a newer version is available.

You could try asking in the [XDA Forum Thread](https://forum.xda-developers.com/t/lineageos-for-microg.3700997/). Someone there might be able to help.

By the way, I'm going to move this issue to the [docker-lineage-cicd issue tracker](https://github.com/lineageos4microg/docker-lineage-cicd/issues) which is what we use for problems with builds - ours or yours. **_This_** issue tracker is only for problems with the extra apps we install
